docs/src/stories/EditableTable.js

```diff
@@ -51,7 +51,10 @@ class Story extends React.PureComponent {
           columns={this.columns}
           defaultPageSize={2}
           showPageSizeOptions={false}
-          showPagination={false}
+          showPagination={{
```

Comment:
I love the idea of allowing both top and bottom pagination, but I'm not a big fan of this syntax. Nested properties in "props" usually introduce weirdness with change detection.  I would like to see an API more like this:

```
showPagination={true}
paginationTop={true} // New prop
paginationBottom={true} // New prop